VERSION

```diff
@@ -1 +1 @@
-22.0509.1
+22.0520.1
```

Comment:
@fasmat FYI I did the version bump / changelog update in this PR, mostly so that I can build the box with a sane version number. Let me know if you want to keep this, or if we should do the version bump / release of new box versions outside of this PR...